python/ray/autoscaler/kubernetes/example-full.yaml

```diff
@@ -95,7 +95,7 @@ head_node_type: head_node
 available_node_types:
   worker_node:
     # Minimum number of Ray workers of this Pod type.
-    min_workers: 0
+    min_workers: 3
     # Maximum number of Ray workers of this Pod type. Takes precedence over min_workers.
     max_workers: 2
```

Comment:
min_workers = 3 and max_workers = 2, this will fail validation. Did you accidentally push this?

Reply:
was experimenting with validation. it doesn't fail. instead it behaves as if min_workers=max_workers=2.
fixed.